sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/AnomalyAlert/AnomalyAlertConfiguration.cs

```diff
@@ -24,22 +34,39 @@ public AnomalyAlertConfiguration(string name, IList<string> idsOfHooksToAlert, I
         }
 
         /// <summary>
+        /// The unique identifier of this <see cref="AnomalyAlertConfiguration"/>. Set by the service.
         /// </summary>
         [CodeGenMember("AnomalyAlertingConfigurationId")]
         public string Id { get; internal set; }
 
         /// <summary>
+        /// A custom name for this <see cref="AnomalyAlertConfiguration"/> to be displayed on fired alerts.
         /// </summary>
         public string Name { get; }
```

Comment:
Not for this PR, but when you say `displayed on`, you mean the only purpose of this property is for display?

I'm asking because in FR we have a property like this and we are using the name `DisplayName` to make the purpose of the property clear for users. We don't want people to use this as an ID or parsing or anything else.

Reply:
> you mean the only purpose of this property is for display

I think so. I'm not 100% sure if we can use it as a filter when listing configurations, but I'd still call it `DisplayName` in this case.

We can only use `Id` for actual identification. I'll add this to the "design considerations" issue.